New Feature: Add images for the wiki and edit links to point to it (#996)
devicons/devicon: README.md

<h2 id="request-icon">Requesting icon</h2>
<p>

>     When you want to request a icon please feel feel to create a issue. See our <a href="https://github.com/devicons/devicon/wiki/Requesting-an-Icon">contribution guidelines</a> for more information.

Reviewer: ```suggestion
    When you want to request an icon please feel free to create an issue. See our <a href="https://github.com/devicons/devicon/wiki/Requesting-an-Icon">Wiki</a> for more information.
```

</p>

<h2 id="contribute">Contributing</h2>
<p>

>     We are happy for every contribution, whether it's new icons, features, or maintainers. Please have a look at our <a href="https://github.com/devicons/devicon/wiki">contribution guidelines</a> to see how you can contribute to this project.

Reviewer: ```suggestion
    We are happy with every contribution, whether it's new icons, features, or maintainers. Please have a look at our <a href="https://github.com/devicons/devicon/wiki">Wiki</a> to see how you can contribute to this project.
```
